addons/account/views/account_account_views.xml

```diff
@@ -60,7 +60,7 @@
                                 <group>
                                     <group>
                                         <field name="account_type" widget="account_type_selection"/>
-                                        <field name="tax_ids" widget="many2many_tags" domain="[('company_id','=',company_id)]" attrs="{'invisible': [('internal_group', '=', 'off_balance')]}"/>
+                                        <field name="tax_ids" widget="many2many_tags" domain="[('company_id','=',company_id)]" attrs="{'invisible': [('internal_group', '=', 'off_balance')]}" options="{'no_quick_create': True}"/>
```

Comment:
no_create should be better. (It is not right place to create taxes, taxes are not created every days :-) )

Reply:
Hi, it is reasonable but, this is not the goal of this task

Reply:
@fmdl It might still be in handy to create taxes in this case.  (Maybe you are setting up everything)  But the thought crossed my mind as well.

Reply:
@jco-odoo @poma-odoo field account_id have already `'no_create':True` https://github.com/odoo/odoo/blob/15.0/addons/account/views/account_move_views.xml#L203, it should be logical that tax are the same behavior.

on all other accounting software it is not possible to create tax on the fly.

Can you do the same things for sale and purchase ?

Reply:
I think we need to keep the things odoo style everywhere, so being able to create is the standard behavior of Odoo and I think we should keep it. Bit Quick create should be removed to prevent accidental creation of meaningless records (where name_create or creating with only name is not possible). Also, in the cases that it action of quick create is opening a form, it does the same thing as create and edit, so we don't need to options that perform the same action.
About sales and purchases, they heave dedicated teams, it is out of my scope of work.

Reply:
For sale order, purchase orders and invoices it makes sense to avoid the create all together because they are daily usage, but these are more config screens.

Reply:
@poma-odoo I don't understand your answer (maybe we said the same things).
Create a taxe is a complexe thing. And I should be create only in the taxes menu (like accounts).